src/main/java/org/jacp/competitionservice/CompetitionserviceApplication.java

```diff
@@ -0,0 +1,13 @@
+package org.jacp.competitionservice;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class CompetitionserviceApplication {
+
+	public static void main(String[] args) {
```

Comment:
Please setup your editor to use 4 spaces instead tabs.

Reply:
done